schemas/answers/checkbox.json

```diff
@@ -0,0 +1,84 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "answer": {
+    "type": "object",
+    "properties": {
+      "id": {
+        "$ref": "../common_definitions.json#/id"
+      },
+      "q_code": {
+        "$ref": "../common_definitions.json#/q_code"
+      },
+      "label": {
+        "type": "string"
+      },
+      "guidance": {
+        "$ref": "../common_definitions.json#/guidance"
+      },
+      "description": {
+        "type": "string"
+      },
+      "type": {
+        "type": "string",
+        "enum": [
+          "Checkbox"
+        ]
+      },
+      "options": {
+        "type": "array",
+        "uniqueItems": true,
+        "items": {
+          "type": "object",
+          "properties": {
+            "label": {
+              "type": "string"
+            },
+            "value": {
+              "type": "string"
+            },
+            "q_code": {
+              "$ref": "../common_definitions.json#/q_code"
+            },
+            "child_answer_id": {
+              "type": "string",
+              "description": "The id of an answer that should be treated as a child of this one, for example a text field answer can be the child of a radio option. The id must be of an answer in this list of answers.",
+              "minLength": 1
+            },
+            "description": {
+              "type": "string",
+              "description": "Descriptive text that appears below the option label"
+            }
+          },
+          "required": [
+            "label",
+            "value"
+          ]
+        }
+      },
+      "mandatory": {
+        "type": "boolean"
+      },
+      "alias": {
+        "type": "string",
+        "description": "An alias that can be used when piping this answer."
```

Comment:
Elements that are common and have a description or restrictions e.g. child_answer_id or alias, should be defined in a common place. This will ensure we don't don't duplicate this information. We might want to think about having something like `schemas/answers/definitions.json` rather than overload the top level definitions.